website/src/components/SimpleTable/SimpleTable.tsx

```diff
@@ -0,0 +1,38 @@
+import { Table, TableContainer, Tbody, Td, Th, Thead, Tr } from "@chakra-ui/react";
+
+interface TableProps {
+  headers: string[];
+  rows: string[][];
+}
+// Create a SimpleTable component that takes in a list of headers and a list of rows
+export const SimpleTable = ({ headers, rows }: TableProps) => {
+  return (
+    <TableContainer>
+      <Table>
+        <Thead>
+          <Tr>
+            {headers.map((header) => (
+              <Th key={header}>
+                {header.split(" ").map((word, index) => (
```

Comment:
Is header not a unique value? We could use css to wrap the text in the heading if its too long.

Reply:
The text of a header can be many words (for instance, "Human messages by language") and needs to be broken in order to have a not so wide table. I don't know how to mix CSS here, so that's why I went with this approach. I'm going to reimplement the table, so maybe this is not needed.

Reply:
I've reimplemented the table, but I still have the problem of the long headers. How do I break them with CSS? I see that there's `word-wrap: break-word;` in CSS, but I don't know how to change the style only for the headers of this table. Could you please give me a pointer?

Reply:
Maybe try this? https://developer.mozilla.org/en-US/docs/Web/CSS/white-space

Reply:
Thank you, but the problem is not the CSS property to use, but that I don't know how to apply CSS to the headers of the DataTable component, which is what is used now for the table. Do you know how to do that?

Reply:
If it's a chakra component you can use the `sx` prop.

Reply:
I've done that: I added the white space property to the DataTable component with the sx prop. It works! Thank you for your help!